test/socket.io.js

```diff
@@ -636,11 +636,10 @@ describe('socket.io', function(){
       });
       function getClients() {
         sio.of('/chat').clients(function(error, sids) {
-          expect(error).to.be.undefined;
```

Comment:
It doesn't matter either way for me. But the value of `error` is actually `null`, so `.to.not.be.ok()` would be better.